### PR TITLE
[#52600] Added drush.yml for drush9.

### PR DESCRIPTION
**Pull request by jibran**
https://redmine.previousnext.com.au/issues/52600
Signed-off-by: Jibran Ijaz <jibran.ijaz@gmail.com>

### CODE REVIEW

File: php/apache/drush.ini [hunk 1]
```diff
@@ -0,0 +1 @@
+memory_limit = 256M
```
Comment (nickschuch) [inline php/apache/drush.ini]: Any way to make this configurable like we used to?

```
$memory = skpr_config('drush.memory') ?: '256M';
ini_set('memory_limit', $memory);
```

Eg. Cron starts to fail in production, you could set that config to temporarily fix the issue.

File: php/apache/Dockerfile [hunk 2]
```diff
@@ -57,6 +57,8 @@ COPY httpd.conf /etc/apache2/httpd.conf
 COPY status.conf /etc/apache2/conf.d/status.conf
 COPY security.conf /etc/apache2/conf.d/security.conf
 COPY drushrc.php /etc/drush/drushrc.php
+COPY drush.yml /etc/drush/drush.yml
+COPY drush.ini /etc/drush/drush.ini
```
Comment (nickschuch) [inline php/apache/Dockerfile]: Is `/etc/drush/drush.ini` a known path? Does drush have docs for this?

File: php/apache/drush.ini (hunk 1 above)
Comment (jibran) [reply php/apache/drush.ini]: Yes!

File: php/apache/Dockerfile (hunk 2 above)
Comment (jibran) [reply php/apache/Dockerfile]: It is a known path but it is not documented anywhere right now. See https://github.com/drush-ops/drush/pull/2506/files#r210795816

Comment (jibran) [reply php/apache/Dockerfile]: Also https://github.com/drush-ops/drush/blob/master/src/Commands/core/EditCommands.php#L120

Comment (jibran) [reply php/apache/Dockerfile]: Output of `drush st`
```
 PHP config       : /etc/php7/php.ini                  
                    /data/drush/drush.ini              
                    /etc/drush/drush.ini
```
```
Drush configs    : /etc/drush/drush.yml               
                    /data/vendor/drush/drush/drush.yml 
                    /data/drush/drush.yml
```

File: php/apache/drush.ini (hunk 1 above)
Comment (acbramley) [reply php/apache/drush.ini]: Can you not pass through this in the env.yml files where we set the uri now?

File: php/apache/Dockerfile (hunk 2 above)
Comment (jibran) [reply php/apache/Dockerfile]: Removed this as we have a new solution to override memory limit.

File: php/apache/drush.ini (hunk 1 above)
Comment (jibran) [reply php/apache/drush.ini]: Removed this file

**Comment by nickschuch** [comment]
Can we keep the drushrc.php for Drupal 7 comparability? Will that cause any issues?

**Comment by nickschuch** [comment]
Add it back and I will merge

**Pull request merged**